modules/admin_manual/pages/installation/deployment_recommendations.adoc

```diff
@@ -96,36 +105,25 @@ One server with at least 2 CPU cores, 16GB RAM, and local storage as needed.
 
 ==== Operating system
 
-Enterprise-grade Linux distribution with full support from an operating system vendor. 
-We recommend both RedHat Enterprise Linux and SUSE Linux Enterprise Server 12.
+Enterprise-grade Linux distribution with full support from an operating system vendor. We recommend Ubuntu 20.04, RedHat Enterprise Linux and SUSE Linux Enterprise Server 12+.
 
 ==== SSL Configuration
 
-The SSL termination is done in Apache. 
-A standard SSL certificate is required to be installed according to {apache-ssl-howto-url}[the official Apache documentation].
+The SSL termination is done in Apache. A standard SSL certificate is required to be installed. See the {apache-ssl-howto-url}[official Apache documentation] or our xref:installation/letsencrypt/index.adoc[Let's Encrypt SSL Certificates] documentation.
 
 ==== Load Balancer
 
 None.
 
 ==== Database
 
-MySQL, MariaDB, or PostgreSQL. 
-We currently recommend MySQL / MariaDB,
-as our customers have had good experiences when moving to a Galera cluster to scale the DB. 
-If using either MySQL or MariaDB, you must use the InnoDB storage engine as MyISAM is not supported, see:
-xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb[MySQL / MariaDB storage engine]
+MySQL, MariaDB, or PostgreSQL. We currently recommend MySQL / MariaDB, as our customers have had good experiences when moving to a Galera cluster to scale the DB. If using either MySQL or MariaDB, you must use the InnoDB storage engine as MyISAM is not supported, see: xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb[MySQL / MariaDB storage engine]
```

Comment:
```suggestion
MySQL, MariaDB, or PostgreSQL. We currently recommend MySQL / MariaDB, as our customers have had good experiences when moving to a Galera cluster to scale the DB. If using either MySQL or MariaDB, you must use the InnoDB storage engine because MyISAM is not supported, see: xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb[MySQL / MariaDB storage engine]
```

Reply:
Just for quicker parsing of the sentence.